src/main/java/Event.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Child class Event inherited from parent class Task
+ */
+public class Event extends Task{
+    public String at;
```

Comment:
Consider renaming this to something else that could distinguish itself from a time or place. See reasons for [renaming](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#not-too-long-not-too-short).